frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -370,98 +360,94 @@ const Card: React.FC<CardProps> = ({ panel }) => {
   );
 };
 
-const Board: React.FC<BoardProps> = ({ board, patchVariable }) => {
-  const [data, setData] = React.useState();
-  const [error, setError] = React.useState<string>();
+const Board: React.FC<BoardProps> = ({ rows }) => (
+  <>
+    {_.map(rows, (row, i) => (
+      <div className="row monitoring-dashboards__row" key={i}>
+        {_.map(row.panels, (panel) => (
+          <Card key={panel.id} panel={panel} />
+        ))}
+      </div>
+    ))}
+  </>
+);
+
+const MonitoringDashboardsPage_: React.FC<MonitoringDashboardsPageProps> = ({
+  clearVariables,
+  deleteAll,
+  match,
+  patchVariable,
+}) => {
+  const [board, setBoard] = React.useState();
+  const [boards, setBoards] = React.useState([]);
+  const [error, setError] = React.useState();
+  const [isLoading, , , setLoaded] = useBoolean(true);
 
   const safeFetch = React.useCallback(useSafeFetch(), []);
 
+  // Clear queries on unmount
+  React.useEffect(() => deleteAll, [deleteAll]);
+
   React.useEffect(() => {
-    if (!board) {
-      return;
-    }
-    setData(undefined);
-    setError(undefined);
-    const path = `${k8sBasePath}/api/v1/namespaces/openshift-monitoring/configmaps/grafana-dashboard-${board}`;
-    safeFetch(path)
+    safeFetch('/api/console/monitoring-dashboard-config')
```

Comment:
We'll need to fix this endpoint for off-cluster mode since we won't have a service account token.

Reply:
> We'll need to fix this endpoint for off-cluster mode since we won't have a service account token.

@kyoto I opened #4407 to make it work off-cluster for development